src/Language/en/Auth.php

```diff
@@ -64,6 +64,12 @@
     'userDoesNotExist'          => 'Password was not changed. User does not exist',
     'resetTokenExpired'         => 'Sorry. Your reset token has expired.',
 
+    // Email Globals
+    'emailInfo'      => 'Some information about the person who asked for the code:',
```

Comment:
Please write in such a way that it covers the `magic link` as well.

Reply:
@datamweb I'd rather add a new line for the magic link.

Reply:
I don't see the need to add a new line on this.
You can use the phrase "Applicant information:" or something similar. Because in `magic-link` we do not send `code`, a `link` is sent.
**However, if you think adding new line would be useful for the Shield, I have no problem with it.**